statsmodels/tsa/automatic/structural.py

```diff
@@ -0,0 +1,659 @@
+        stochastic_seasonal = []
+        stochastic_cycle = []
+        damped_cycle = []
+        measure = []
+        for res in self.values():
+            spec = res.specification
+
+            level.append(spec.level)
+            seasonal.append(spec.seasonal)
+            cycle.append(spec.cycle)
+            autoregressive.append(spec.autoregressive)
+            stochastic_seasonal.append(spec.stochastic_seasonal)
+            stochastic_cycle.append(spec.stochastic_cycle)
+            damped_cycle.append(spec.damped_cycle)
+            measure.append(res.value)
+
+        # Add the model specification information
+        param_header = ['level']
+        params_data = [level]
+        if np.any(seasonal):
+            param_header.append('seasonal')
+            params_data.append(seasonal)
+        if np.any(cycle):
+            param_header.append('cycle')
+            params_data.append(cycle)
+        if np.any(autoregressive):
+            param_header.append('autoregressive')
+            params_data.append(autoregressive)
+        if np.any(seasonal) and np.any(stochastic_seasonal):
+            param_header.append('stochastic_seasonal')
+            params_data.append(stochastic_seasonal)
+        if np.any(cycle):
+            if np.any(stochastic_cycle):
+                param_header.append('stochastic_cycle')
+                params_data.append(stochastic_cycle)
+            if np.any(damped_cycle):
+                param_header.append('damped_cycle')
+                params_data.append(damped_cycle)
+
+        # Add the specified measure
+        param_header.append(self.measure)
+        params_data.append(['%#6.3f' % x for x in measure])
+
+        # Add the other measures, if applicable
+        for other_measure in ['aic', 'bic', 'hqic']:
+            if other_measure == self.measure:
+                continue
+            param_header.append(other_measure)
+            params_data.append(['%#6.2f' % getattr(x.results, other_measure, None)
+                                for x in self.values()
+                                if x.results is not None])
+
+        # Add the number of parameters, if applicable
+        param_header.append('k_params')
+        params_data.append(['%d' % len(getattr(x.results, 'params'))
+                            for x in self.values()
+                            if x.results is not None])
+
+        # Add in the status
+        param_header.append('warnings/exception')
+        params_data.append(['%s' % 'No' if x.status == 0 else 'Yes'
+                            for x in self.values()])
+
+        params_data = list(zip(*params_data))
+        params_stubs = ['%d ' % x for x in range(len(params_data))]
+
+        from statsmodels.iolib.table import SimpleTable
+        models_table = SimpleTable(params_data,
+                                      param_header,
+                                      params_stubs,
+                                      title='Evaluated models',
```

Comment:
travis failures are pep-8, e.g. overindended line

Reply:
Once addressed,  please add all these new files to lint.sh